### ISSUE TITLE
Create a Form Element

### ISSUE THREAD

**Issue by GT1man** [freeCodeCamp/freeCodeCamp#14890]
I am on the Create a Form Element challenge. Challenge number 36. The iPhone to the right does not display anything aside from an error after I click and begin typing or deleting or even just clicking anywhere. The code there was code that the computer added automatically after a challenge was completed. None of the code in the picture was added by me. I have closed out of the window, reloaded it, restarted my computer and the issue is still happening. When I delete any code or add any code the issue still persists. I am not able to get rid of the error no matter what I do. I have cleared everything in there, I have hit the reset button under "run tests". The same thing still persists.

I am using Google Chrome on an Asus C300 Chromebook running Chrome OS. I know chrome os isn't normally supported but it has been working up until this point. As far as I know it is up to date on the latest software. It has not alerted me there is an update available.

![screenshot 2017-05-16 at 12 01 58 am](https://cloud.githubusercontent.com/assets/28659255/26091936/6f902d3a-39cb-11e7-9ec1-b6178390d658.png) 
Before I click anywhere

![screenshot 2017-05-15 at 11 53 43 pm](https://cloud.githubusercontent.com/assets/28659255/26091963/90a18a64-39cb-11e7-8aa8-ecb75312321b.png)
After I click anywhere

**Comment by raisedadead** [comment]
Hi @GT1man 

Thanks for reporting this issue. 

Here are some next steps that you should follow before we can help you with your query:
- Check [these troubleshooting steps](https://forum.freecodecamp.com/t/how-to-report-a-bug/19543) first.
- Please use the [Chat Rooms](https://forum.freecodecamp.com/t/free-code-camp-official-chat-rooms/19390) or try looking through our [forum](http://forum.freecodecamp.com) for help with a specific challenge and any coding assistance.

**IMPORTANT:** 
Please NOTE that this tracker is for reporting bugs and enhancements to freeCodeCamp's code base only.

Happy Coding!

**Comment by raisedadead** [comment]
FYI Challenges are not numbered. Those figures besides your profile are your scores, more info here https://forum.freecodecamp.com/t/free-code-camp-brownie-points/18380

**Comment by UnklePennybags** [comment]
Did you reset the code and then try clicking test again to see if the issue is resolved? also if you double click to fast some hardware may cause it to not function properly and display a blank screen with an error msg.

**Comment by GT1man** [comment]
I reset the code, posted in a forum but still haven't received any useful information back yet. And it is still happening.

**Comment by raisedadead** [comment]
Duplicate of #13727 
Please use firefox while we try and come up with a fix.
